src/conn/conn_dhandle.c

```diff
@@ -108,6 +114,14 @@ __conn_dhandle_config_set(WT_SESSION_IMPL *session)
     }
     dhandle->cfg[1] = metaconf;
     dhandle->meta_base = base;
+    dhandle->orig_meta_base_length = base == NULL ? 0 : strlen(base);
```

Comment:
Minor comment: I would name this `meta_base_length` because the prefix `orig_` makes it seem like it is related to the new `orig_meta_base` you added.